### DISCUSSION TITLE
vulnerabilities page - aliases column is empty

### DISCUSSION THREAD

**Discussion by setchy** [DependencyTrack/dependency-track#2939] [Q&A]
Is the aliases column shown on `https://some-dtrack-host/vulnerabilities` meant to be populated with values?  

I've found that in our install this column is always empty, but when i drill into a specific CVE it will show GHSA aliases etc.

**Comment by nscuro** [comment]
I could've sworn this was logged as defect before, and fixed in a previous version...

That behavior is defo not intended and would be a bug.

**Reply by setchy** [reply]
I closed this discussion this morning as outdated as I had the same feeling :)

**Reply by nscuro** [reply]
Whoops, didn't look at the date and the notification on GitHub mobile made it look like a new post. Nevermind then. 😆
